compilers/concrete-compiler/compiler/.gitignore

```diff
@@ -15,4 +15,3 @@ tests/end_to_end_fixture/tests_cpu/end_to_end_apply_lookup_table.yaml
 tests/end_to_end_fixture/tests_cpu/end_to_end_leveled.yaml
 tests/end_to_end_fixture/tests_cpu/end_to_end_linalg_apply_lookup_table.yaml
 tests/end_to_end_fixture/tests_cpu/end_to_end_multi_precision.yaml
-
```

Comment:
not clear in github view if the line 17 still has a end of line at the end

Reply:
is should be checked by the linelint